precompiles/xvm/src/tests.rs

```diff
@@ -50,6 +51,7 @@
                     .write(0u8)
                     .write(Bytes(b"".to_vec()))
                     .write(Bytes(b"".to_vec()))
+                    .write(U256::one())
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A `U256::one()` value is being written, but it seems there's no context provided for the purpose of this write. If it's related to a specific test case or expected value, ensure that the comment or code snippet provides clear context and explanation. If this is a mistake or incomplete code, address it accordingly.

```diff
                    .write(U256::one())
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->